definitions/features/foreman_proxy.rb

```diff
@@ -27,9 +38,9 @@ def services
   private
 
   def dhcp_curl_cmd
-    "curl -w '\n%{http_code}' --silent -ks --cert #{cert_path}/client_cert.pem \
-      --key #{cert_path}/client_key.pem \
-      --cacert #{cert_path}/proxy_ca.pem https://$(hostname):9090/dhcp"
+    "curl -w '\n%{http_code}' -s --cert #{proxy_settings[:foreman_ssl_cert]} \
+      --key #{proxy_settings[:foreman_ssl_key]} \
```

Comment:
This should default to `proxy_settings[:ssl_{certificate,private_key,ca_file}]` if ithe `foreman_*` one are not specified.

Reply:
I actually wonder if it shouldn't always default to the non-foreman ones. They should be signed by the same CA as it's listening on so they should always be accepted. Then it is important that the host itself is in trusted hosts, but IIRC the installer does that by default.

Foreman can have a custom CA (say letsencrypt) so then the CA verification would fail. cert/key could also be different but that's less likely.

Reply:
If we didn't used primarily the `foreman_*` one, we would actually break the custom-cert case. The fact that in many cases, this two are the identical is just a lucky coincidence (connected to the fact, that we are using the client certs also as server certs, but we should not make this assumption IMO)